packages/top-nav/src/TopNav.ts

```diff
@@ -169,17 +169,11 @@ export class TopNav extends SizedMixin(SpectrumElement) {
             document.fonts ? document.fonts.ready : Promise.resolve(),
         ]);
         const itemBoundingClientRect = selectedItem.getBoundingClientRect();
-        const parentBoundingClientRect = this.getBoundingClientRect();
 
         const width = itemBoundingClientRect.width;
-        const offset =
-            this.dir === 'ltr'
-                ? itemBoundingClientRect.left - parentBoundingClientRect.left
-                : itemBoundingClientRect.right - parentBoundingClientRect.right;
-
-        this.selectionIndicatorStyle = `transform: translateX(${offset}px) scaleX(${
-            this.dir === 'ltr' ? width : -1 * width
-        });`;
+        const offset = selectedItem.offsetLeft;
```

Comment:
I dig the simplification. Does `offsetLeft` inherently measure from the right in `rtl` contexts?

Reply:
I actually hoped that there would be an `offsetRight` and I could just swap, but `offsetLeft` [_always_ measuring from left](https://codepen.io/Westbrook/pen/abYYvMe), meant that all of this logic could be simplified. A kind of crazy side effect of the whole deal!